SCClassLibrary/Common/Collections/SequenceableCollection.sc

```diff
@@ -914,8 +914,8 @@ SequenceableCollection : Collection {
 
 	rate {
 		var rate, rates;
-		if(this.size == 1, { ^this.first.rate });
-		^this.collect({ arg item; item.rate }).minItem;
+		if(this.size == 1) { ^this.first.rate };
+		^this.collect { arg item; item.rate ? 'scalar' }.minItem;
```

Comment:
style-wise I much pref this.collect({ ... }).minItem

{  }.minItem looks like a method on the function.  but its actually a method on the result of this.collect({ })

so it requires some esoteric knowledge about the many ways that sc will apply sugar, just to understand why the code works here.